toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -380,6 +380,9 @@ build_rpm_in_chroot_no_install curl
 chroot_and_install_rpms python-setuptools
 build_rpm_in_chroot_no_install cracklib
 
+# pam needs cracklib
+chroot_and_install_rpms cracklib
```

Comment:
Could we also move this to `1.0-dev`?

Reply:
Will do once `dev` completes a full pipeline build.